src/test/java/org/sagebionetworks/bridge/sdk/integration/AuthenticationTest.java

```diff
@@ -22,7 +21,6 @@
 import org.sagebionetworks.bridge.sdk.models.studies.Study;
 import org.sagebionetworks.bridge.sdk.models.users.EmailCredentials;
 import org.sagebionetworks.bridge.sdk.models.users.SignInCredentials;
-import org.sagebionetworks.bridge.sdk.models.users.SignUpByAdmin;
 import org.sagebionetworks.bridge.sdk.models.users.SignUpCredentials;
 
 @Category(IntegrationSmokeTest.class)
```

Comment:
These changes don't seem to be related to the others--just confirming they're intentional?